tests/lib.rs

```diff
@@ -133,6 +133,7 @@ impl Arbitrary for Proto {
                 Proto(Onion3((a, std::cmp::max(1, u16::arbitrary(g))).into()))
             }
             25 => Proto(Tls),
+            26 => Proto(QuicV1),
```

Comment:
We are missing some protocols, see https://github.com/multiformats/rust-multiaddr/issues/65. Though that is out of scope for this pull request.